rework_tests/unit/estimators/iterative/_strategies/test_observed_data_likelihood.py

```diff
@@ -0,0 +1,257 @@
+"""Tests for Observed Data Likelihood optimization strategy."""
+
+__author__ = "Danil Totmyanin"
+__copyright__ = "Copyright (c) 2025 PySATL project"
+__license__ = "SPDX-License-Identifier: MIT"
+
+
+import numpy as np
+import pytest
+from rework_pysatl_mpest.core import MixtureModel, Parameter
+from rework_pysatl_mpest.distributions import ContinuousDistribution
+from rework_pysatl_mpest.estimators.iterative import MaximizationStrategy, OptimizationBlock, PipelineState
+from rework_pysatl_mpest.estimators.iterative._strategies import observed_data_likelihood_strategy
+from rework_pysatl_mpest.optimizers import Optimizer
+
+DTYPES_TO_TEST = [np.float32, np.float64]
```

Comment:
```suggestion
DTYPES_TO_TEST = [np.float16, np.float32, np.float64]
```
All tests passed successfully.